src/tests/Microsoft.PowerFx.Interpreter.Tests/TextDateTimeToUTCTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Timers;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class TextDateTimeToUTCTests : PowerFxTest
+    {
+        [Theory]
+        [InlineData("Text(DateTimeValue(\"March 10, 2023 5:30 PM\"), DateTimeFormat.UTC)", "Pacific Standard Time", "2023-03-11T01:30:00Z")]
```

Comment:
>00 [](http://example.com/codeflow?start=134&length=2)

The format in Power Apps uses 3 decimal places (i.e., `"2023-03-11T01:30:00.000Z"`). Date/Time values in C# have 7